index.html

```diff
@@ -17,6 +17,7 @@
             <li class="contact">Contact</li>
```

Comment:
- [ ] It would be better if we can start the application without any errors and successfully. After following the instructions in the README here is the result in the screenshot below
![Screenshot from 2022-03-23 20-18-49](https://user-images.githubusercontent.com/46388113/159789249-b74934d9-9e62-4365-b699-c5cb5966c770.png)

Reply:
The error says you need to install `http-server`  not found. You need to  install it globally using ` npm install http-server -g`